.make/docker.mk

```diff
@@ -0,0 +1,122 @@
+DOCKER_IMAGE_CORE := $(PROJECT_NAME)
```

Comment:
should you include the `latest` tag here ?

Reply:
It is applied here:
https://github.com/fabric8-services/fabric8-toggles-service/pull/8/files#diff-9f1c1c4a84a9bc024479d52a8218fa0fR38 (note only the deployment image is pushed to docker hub)

Reply:
ok